### PR TITLE
Sapdrm 17 docker install

### PR DESCRIPTION
**Pull request by RajeshwariMakond**
Added docker install cookbook

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # cookbooksrepo
 
 This repo is for rean assignments.
+Added Hashicorp Consul cookbook
```
Comment (aashishmodak) [inline README.md]: This is not consul, change this to docker

File: docker-install/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# docker-install
+
+TODO: Enter the cookbook description here.
```
Comment (aashishmodak) [inline docker-install/README.md]: add proper description, refer readme from https://github.com/miketheman/nginx/tree/v2.6.0

File: docker-install/metadata.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name 'docker-install'
+maintainer 'The Authors'
```
Comment (aashishmodak) [inline docker-install/metadata.rb]: add your name as maintainer

File: docker-install/metadata.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+maintainer_email 'you@example.com'
```
Comment (aashishmodak) [inline docker-install/metadata.rb]: add your email as maintainer email

File: docker-install/metadata.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+description 'Installs/Configures docker-install'
+long_description 'Installs/Configures docker-install'
+version '0.1.0'
+chef_version '>= 12.1' if respond_to?(:chef_version)
+
+# The `issues_url` points to the location where issues for this cookbook are
+# tracked.  A `View Issues` link will be displayed on this cookbook's page when
+# uploaded to a Supermarket.
+#
+# issues_url 'https://github.com/<insert_org_here>/docker-install/issues'
```
Comment (aashishmodak) [inline docker-install/metadata.rb]: update issue url

File: docker-install/metadata.rb [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+# The `source_url` points to the development repository for this cookbook.  A
+# `View Source` link will be displayed on this cookbook's page when uploaded to
+# a Supermarket.
+#
+# source_url 'https://github.com/<insert_org_here>/docker-install'
```
Comment (aashishmodak) [inline docker-install/metadata.rb]: update source url

**Review by aashishmodak — CHANGES_REQUESTED** [review]
Fix the review comments and merge conflict

**Comment by RajeshwariMakond** [comment]
Resolved Conflict. Please review

File: docker-install/README.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+# docker-install
+
+docker-install Cookbook
+=======================
+Installs Hashicorp vault from package OR source code to CentOS 7
```
Comment (aashishmodak) [inline docker-install/README.md]: This is not for Vault

File: docker-install/README.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+# docker-install
+
+docker-install Cookbook
+=======================
+Installs Hashicorp vault from package OR source code to CentOS 7      
+
+Requirements
+------------
+Installs Hashicorp vault on CentOS 7
```
Comment (aashishmodak) [inline docker-install/README.md]: Please correct the readme

**Review by aashishmodak — CHANGES_REQUESTED** [review]
Please correct readme

**Review by aashishmodak — APPROVED** [review]

**Pull request merged**